deploy/registration-service.yaml

```diff
@@ -47,6 +47,13 @@ objects:
           - toolchain.dev.openshift.com
         resources:
           - bannedusers
+        verbs:
```

Comment:
minor thing - but since you decoupled the update of the deployment manifest in a separate PR in host-operator I would create a dedicated PR for this so that it can be merged together with the host one. WDYT?

Reply:
never mind - the manifest was dropped from this repo in this PR https://github.com/codeready-toolchain/registration-service/pull/459